server.js

```diff
@@ -10,7 +10,7 @@ const scriptsToLoad = process.env.SCRIPTS || './scripts/**/*.js'
 const { app, events } = require('./app')
 
 // load all the files in the scripts folder
-globSync(scriptsToLoad).forEach((file) => {
+globSync(scriptsToLoad, { dotRelative: true }).forEach((file) => {
   logger.info('Loading:', file)
   require(file)(app, events)
```

Comment:
I don't know why it's not showing up in this diff, but this line is currently:

```
require(`./${file}`)(app, events)
```

and will need to be adjusted.

Reply:
Maybe rebase the pr branch against main?

Reply:
Rebased and updated.